### ISSUE TITLE
Report which AP is currently in use

### ISSUE THREAD

**Issue by fhunleth** [nerves-networking/vintage_net#99]
It's possible to setup multiple access points, but it's not reported anywhere which access point is actually in use. This would be nice to know.

**Comment by fhunleth** [comment]
This ignored event looks helpful:

```
04:44:42.555 [info]  WPASupplicant ignoring {:event, "CTRL-EVENT-CONNECTED", "18:e8:29:17:fe:74", "completed", %{"id" => "0"}}
```

If haven't seen an event when the AP goes away yet...

**Comment by mattludwigs** [comment]
Would the information get from these events be stored in the property table?

**Comment by fhunleth** [comment]
Yeah, I think that makes the most sense.

**Comment by fhunleth** [comment]
Fixed in b72845667e07f89975cb4807cb8f84e9bd42a4ce.
